model-api-gen-runtime/src/commonMain/kotlin/org/modelix/metamodel/ITypedProperty.kt

```diff
@@ -27,3 +27,28 @@ fun <ValueT> INode.setTypedPropertyValue(property: ITypedProperty<ValueT>, value
 fun <ValueT> INode.getTypedPropertyValue(property: ITypedProperty<ValueT>): ValueT {
     return property.deserializeValue(getPropertyValue(property.untyped()))
 }
+
+fun INode.getBooleanPropertyValue(property: IProperty): Boolean {
```

Comment:
nit: In public APIs like this, we wanted to document what exceptions are thrown in which circumstances.

Reply:
Dropped that change.